readme.md

```diff
@@ -1,54 +1,36 @@
 # Dynatrace Azure Log Forwarder
-## Overview
-This project provides a mechanism that allows streaming Azure logs from Azure Event Hub into Dynatrace Logs via Azure Function App. 
-It supports both: Azure Resource Logs and Azure Activity Logs.
-
-Prerequisites:
-* Azure CLI - if you run deployment script from your machine (not needed when using Azure Portal Cloud Shell) [https://docs.microsoft.com/en-us/cli/azure/install-azure-cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
-* running Azure Event Hub(s) instance(s) in each Azure location you want to pull logs from 
-* configured `diagnostic settings` for resources you would like to stream logs from, pointing to Even Hub(s) (reference [https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings))
-
-
-*Architecture*
-
-azure-log-forwarder-function is an Azure Function App written in Python that pulls logs for configured services. The Function execution is triggered by Azure Event Hub. Once all log records are processed, they are pushed to Dynatrace Logs API.   
-
-`azure-log-forwarder` (with containerized ActiveGate)  components:
-* `azure-log-forwarder-function` - function app that subscribes to one Event Hub which is in the same location where you deploy azure-log-forwarder; deployment with App Service Plan and minimum `S1 Standard` pricing tier 
-* `azure-log-forwarder-container` - container instances that provides Dynatrace ActiveGate proxy to Dynatrace Infrastructure. This is Azure Container Instance with 1 core, 1.5 GB ram assigned
-* `azure-log-forwarder-vnet` - virtual network to ensure private communication between function and active-gate
-
-You need to deploy azure-log-forwarder in each region you want to pull logs from.
-
-![Architecture](./img/architecture.png)
+# Overview
+ This project provides a mechanism that allows streaming Azure logs from Azure Event Hub into Dynatrace Logs via Azure Function App. It supports both: Azure Resource Logs and Azure Activity Logs.
 
 
+# Prerequisites
+Please refer to [Prerequisites](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#prerequisites-) Documentation.
+ 
 # Deployment
-Please refer to the instructions:  
-   [Dynatrace log forwarder Documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd)
+Please refer  to [Dynatrace log forwarder](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd) Documentation.
 
 # Viewing Azure logs in Dynatrace UI
- Please [refer to documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#view-azure-logs).
+Please refer to [documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#view-azure-logs).
```

Comment:
Inconsistent Documentaion /documentation convention. Additionally I miss the documentation title here.

Reply:
https://github.com/dynatrace-oss/dynatrace-azure-log-forwarder/pull/18#discussion_r648216615